Makefile

```diff
@@ -16,7 +16,7 @@ CC1       := tools/agbcc/bin/agbcc$(EXE)
 CPP       := $(PREFIX)cpp
 LD        := $(PREFIX)ld
 OBJCOPY   := $(PREFIX)objcopy
-SHA1SUM   := sha1sum -c
```

Comment:
Or just add `brew install md5sha1sum` to the macOS setup instructions

Reply:
Is there a preference for one change over the other?

Reply:
I think the best possible solution to this would be to let the end-user choose between what's documented here in this PR and `brew install`ing `md5sha1sum`.

Reply:
That will always be the case. This doesn't prevent people from installing `md5sha1sum`